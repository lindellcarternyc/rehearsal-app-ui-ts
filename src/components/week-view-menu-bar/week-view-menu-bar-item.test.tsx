import * as React from 'react'
import * as renderer from 'react-test-renderer'
import * as Enzyme from 'enzyme'
import * as Adapter from 'enzyme-adapter-react-16'

import WeekViewMenuBarItem from './week-view-menu-bar-item'

import WEEKS from '../../mocks/data/weeks'

Enzyme.configure({adapter: new Adapter()})

describe('WeekViewMenuBarItem', () => {
  it('renders with no rehearsals', () => {
    const day = WEEKS[0].days[0]
    const tree = renderer.create(
      <WeekViewMenuBarItem day={day} />
    )
    expect(tree).toMatchSnapshot()
  })

  it('renders with rehearsals', () => {
    const day = WEEKS[1].days[2]
    const tree = renderer.create(
      <WeekViewMenuBarItem day={day} />
    )
    expect(tree).toMatchSnapshot()
  })
})