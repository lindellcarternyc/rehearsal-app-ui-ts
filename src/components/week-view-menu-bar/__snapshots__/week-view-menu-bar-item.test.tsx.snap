// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeekViewMenuBarItem renders with no rehearsals 1`] = `
<div
  className="ui list"
  role="list"
>
  <div
    className="item"
    data-value={undefined}
    onClick={[Function]}
    role="listitem"
  >
    <div
      className="description"
    >
      S
    </div>
    <h3
      className="header"
    >
      7
    </h3>
    <i
      aria-hidden="true"
      className="grey circle tiny icon"
    />
  </div>
</div>
`;

exports[`WeekViewMenuBarItem renders with rehearsals 1`] = `
<div
  className="ui list"
  role="list"
>
  <div
    className="item"
    data-value={undefined}
    onClick={[Function]}
    role="listitem"
  >
    <div
      className="description"
    >
      T
    </div>
    <h3
      className="header"
    >
      16
    </h3>
    <i
      aria-hidden="true"
      className="blue circle tiny icon"
    />
  </div>
</div>
`;
